app/experimenter/nimbus-ui/src/components/App/index.test.tsx

```diff
@@ -51,15 +51,15 @@ describe("App", () => {
       const {
         history: { navigate },
       } = renderWithRouter(
-        <MockedCache mocks={[mock]}>
+        <MockedCache mocks={[mock, mock]}>
           <App basepath="/" />
         </MockedCache>,
         {
           route: `/my-special-slug/`,
         },
       );
 
-      await navigate("/my-special-slug/edit");
+      await act(() => navigate("/my-special-slug/edit"));
```

Comment:
Not sure if some related package was recently updated or what, but I was getting a complaint here about not wrapping this in `act` which makes sense as all other `navigate` instances use this.

Reply:
I haven't seen any test complaints about this, but makes sense.